composer.json

```diff
@@ -21,7 +21,7 @@
     "require": {
         "php": "^7.3",
 
-        "sylius/sylius": "^1.7",
+        "sylius/sylius": "^1.7|^1.8",
```

Comment:
Probably we should test the plugin on both Sylius 1.7 and 1.8 (similar as it is in [RefundPlugin](https://github.com/Sylius/RefundPlugin/blob/master/.github/workflows/build.yml#L114))